src/JustSaying/Fluent/TopicPublicationBuilder`1.cs

```diff
@@ -29,6 +30,11 @@ internal TopicPublicationBuilder()
         /// </summary>
         private Action<SnsWriteConfiguration> ConfigureWrites { get; set; }
 
+        /// <summary>
+        /// Gets the tags to add to the topic.
+        /// </summary>
+        private Dictionary<string, string> Tags { get; } = new(StringComparer.OrdinalIgnoreCase);
```

Comment:
Same here - the public API would be more lenient to allow an `IDictionary<string, string>` rather than enforce a specific concrete type.

Reply:
This is only added to as a result of calls to WithTag so would changing it be of any benefit here?